gapii/cc/spy.cpp

```diff
@@ -262,7 +263,15 @@ std::shared_ptr<StaticContextState> GlesSpy::GetEGLStaticContextState(CallObserv
 
     Constants constants;
     getContextConstants(constants);
-    std::shared_ptr<StaticContextState> out(new StaticContextState(constants));
+
+    std::string threadName;
+#if TARGET_OS == GAPID_OS_ANDROID
+    char buffer[256] = { 0 };
```

Comment:
I noticed, but the size is not passed down to the getter function.
I am thinking better safe then sorry.